docker/docker-compose.yml

```diff
@@ -49,6 +49,11 @@ services:
 
   web:
     <<: *takahe-common
+    healthcheck:
+      test: ["CMD", "nc", "-z", "-v", "localhost", "8000"]
```

Comment:
Is this health check essentially "connect to the port, but don't do anything with the connection"? If so, curl to `/actor/` would let us know that both nginx & takahe are running well enough to return a vital (fast) endpoint.

Reply:
It tells docker, that the service is operating healthy or not. With `docker ps` it would look like this:  
![](https://mws.bka.li/sharex/2022/12/kitty-0.76.0.8_20221227_062247.png)

But yes, one could connect to nginx too, but I thought I keep it out for the moment because of https://github.com/jointakahe/takahe/pull/278 which splits some things up.